samples/oauth_pkce_cli/pkce-cli

```diff
@@ -10,7 +10,7 @@ var opn = require('opn');
 
 program
   .option('-c, --client_id <client id>', 'OIDC Client ID', '')
-  .option('-a, --auth_server <auth server url>', 'ex: https://sandbox-api.va.gov', '')
+  .option('-a, --auth_server <auth server url>', 'ex: https://sandbox-api.va.gov/oauth2/veteran-verification-apis/v1', '')
```

Comment:
There are several references in README.md that should also be updated.
- [ ] Line 34
- [ ] Line 52
- [ ] Line 64